path_creds_create.go

```diff
@@ -62,7 +60,7 @@ func (b *backend) credsReadHandler(ctx context.Context, req *logical.Request, d
 	if err != nil {
 		return nil, err
 	}
-	username := fmt.Sprintf("vault_%s_%s_%s_%d", name, req.DisplayName, userUUID, time.Now().UnixNano())
+	username := fmt.Sprintf("%s_%s", role.UserPrefix, userUUID)
```

Comment:
userUUID might not be unique, e.g., did you test same user requesting two emphemeral creds with overlapping lifetime?

Reply:
also, should we keep requestor name in there for tracking? (at least unless explicitly overridden)